src/api/clients/IOClients/apps/Billing.ts

```diff
@@ -14,10 +14,13 @@ export class Billing extends AppClient {
   public installApp = async (
     appName: string,
     termsOfUseAccepted: boolean,
-    force: boolean
+    force: boolean,
+    selectedPlanId?: string
   ): Promise<InstallResponse> => {
     const graphQLQuery = `mutation InstallApps{
-      install(appName:"${appName}", termsOfUseAccepted:${termsOfUseAccepted}, force:${force}) {
+      install(appName:"${appName}", termsOfUseAccepted:${termsOfUseAccepted}, force:${force}${
+      selectedPlanId ? `, selectedPlanId: "${selectedPlanId}"` : ''
+    }) {
```

Comment:
Is this going to make any difference?

Reply:
No, it's not! Removed in https://github.com/vtex/toolbelt/pull/1092/commits/3bb180e8ddc4b09fa965cf0c6e39fa29894196a2

Thanks 😁